xforce.json

```diff
@@ -1,16 +1,16 @@
 {
     "appid": "653a4735-1eed-4cb9-98c4-b40408e60b22",
-    "name": "IBM XForce",
-    "description": "This app implements various 'investigative' actions on the 'IBM XForce' device",
+    "name": "IBM X-Force Exchange",
+    "description": "This app implements various 'investigative' actions on the 'IBM X-Force Exchange' device",
     "publisher": "AvantGarde Partners",
-    "package_name": "phantom_xforce",
+    "package_name": "phantom_ibmx-forceexchange",
     "type": "investigative",
-    "license": "Copyright (c) AvantGarde Partners 2017",
+    "copyright": "Copyright (c) AvantGarde Partners 2017",
     "main_module": "xforce_connector.py",
     "app_version": "1.1.0",
     "utctime_updated": "2022-01-07T20:46:01.000000Z",
     "product_vendor": "IBM",
```

Comment:
@mattsayar-splunk I also want to mention that I'm keeping the `product_vendor` as `IBM` here.